genie-agent/src/test/groovy/com/netflix/genie/agent/cli/ArgumentParserSpec.groovy

```diff
@@ -19,8 +19,11 @@
 package com.netflix.genie.agent.cli
 
 import com.beust.jcommander.JCommander
+import com.netflix.genie.test.categories.UnitTest
+import org.junit.experimental.categories.Category
 import spock.lang.Specification
 
+@Category(UnitTest.class)
```

Comment:
Have we tested if this categorization even works with spock tests? I think it does cause spock builds on junit but just a random question if we're wasting our time putting this

Reply:
It works